en/computer_vision/collision_prevention.md

```diff
@@ -130,16 +127,49 @@ If the vehicle feels 'stuck' with only a single distance sensor pointing forward
 
 ## PX4 Distance Sensor
 
+### Lanbao PSK-CM8JL65-CC5
+
 At time of writing PX4 allows you to use the [Lanbao PSK-CM8JL65-CC5](../sensor/cm8jl65_ir_distance_sensor.md) IR distance sensor for collision prevention "out of the box", with minimal additional configuration:
+
 - First [attach and configure the sensor](../sensor/cm8jl65_ir_distance_sensor.md), and enable collision prevention (as described above, using [CP_DIST](#CP_DIST)).
 - Set the sensor orientation using [SENS_CM8JL65_R_0](../advanced_config/parameter_reference.md#SENS_CM8JL65_R_0).
 
+### LightWare LiDAR SF45 Rotating Lidar
+
+PX4 v1.14 (and later) supports the [LightWare LiDAR SF45](https://www.lightwarelidar.com/shop/sf45-b-50-m/) rotating lidar which provides 320 degree sensing.
+
+The SF45 must be connected via a UART/serial port and configured as described below (In addition to the [collision prevention setup](#px4-software-setup)).
+
+[LightWare Studio](https://www.lightwarelidar.com/resources-software) configuration:
+
+- In the LightWare Studio app enable scanning, set the scan angle, and change the baud rate to `921600`.
+
+PX4 Configuration:
+
+- Add the [lightware_sf45_serial](../modules/modules_driver_distance_sensor.md#lightware-sf45-serial) driver in [menuconfig](../hardware/porting_guide_config.md#px4-menuconfig-setup):
+  - Under **drivers > Distance sensors** select `lightware_sf45_serial`.
+  - Recompile and upload to the flight controller.
+- [Set the following parameters](../advanced_config/parameters.md) via QGC:
+  - [SENS_EN_SF45_CFG](../advanced_config/parameter_reference.md#SENS_EN_SF45_CFG): Set to the serial port you have the sensor connected to.
+    Make sure GPS or Telemetry are not enabled on this port.
+  - [SF45_ORIENT_CFG](../advanced_config/parameter_reference.md#SF45_ORIENT_CFG): Set the orientation of the sensor (facing up or down)
+  - [SF45_UPDATE_CFG](../advanced_config/parameter_reference.md#SF45_UPDATE_CFG): Set the update rate
+  - [SF45_YAW_CFG](../advanced_config/parameter_reference.md#SF45_YAW_CFG): Set the yaw orientation
+
+In QGroundControl you should see an [OBSTACLE_DISTANCE](https://mavlink.io/en/messages/common.html#OBSTACLE_DISTANCE) message in the [MAVLink console](../debug/mavlink_shell.md#qgroundcontrol-mavlink-console) if collision prevention is configured correctly and active.
+
+The obstacle overlay in QGC will look like this:
+
+![sf45](../../assets/sf45/sf45_obstacle_map.png)
```

Comment:
FYI, I've modified this to better cross link to other relevant information that might not be so well understood by some users - such as menuconfig. Also to make it a little more readable. The core of it is the same.

Reply:
PS But a quick scan to check you are still happy with this would be good!

Reply:
Looks good, thanks